### PR TITLE
ACM-7909: add a note about uniqueness of hosted cluster names

### PR DESCRIPTION
**Pull request by rokej**
This is for https://issues.redhat.com/browse/ACM-7563, https://issues.redhat.com/browse/ACM-7909.

We need to add a note about MCE's requirement that each hosted cluster must have a unique name to be managed by MCE or ACM. 

Also a correction for BareMetal that it is not tech-preview any more.

### CODE REVIEW

**Comment by rokej** [comment]
@lahinson Could you please refine and format my updates in this PR? Thank you.

File: clusters/hosted_control_planes/managing_hosted_aws.adoc [hunk 1]
```diff
@@ -1,7 +1,10 @@
 [#hosted-control-planes-manage-aws]
 = Managing hosted control plane clusters on AWS (Technology Preview)
 
-You can use the {mce} console to create a {ocp} hosted cluster. If you use hosted control planes on AWS, you can create a hosted cluster by using the console, or you can import a hosted cluster by using either the console or the command line interface.
+You can use the hcp command line interface to create a {ocp} hosted cluster. The hosted cluster is automatically imported as a managed cluster. If you wish to disable this automatic import feature, refer to https://github.com/stolostron/rhacm-docs/blob/2.9_stage/clusters/hosted_control_planes/hosted_disable_auto_import.adoc.
```
Comment (lahinson) [inline clusters/hosted_control_planes/managing_hosted_aws.adoc]:
```suggestion
You can use the hosted control plane command line interface (`hcp`) to create an {ocp-short} hosted cluster. The hosted cluster is automatically imported as a managed cluster. If you want to disable this automatic import feature, see xref:../hosted_control_planes/hosted_disable_auto_import.adoc#hosted-disable-auto-import[Disabling the automatic import of hosted clusters into {mce-short}].
```

File: clusters/hosted_control_planes/managing_hosted_aws.adoc [hunk 2]
```diff
@@ -1,7 +1,10 @@
...
+
+Note: Each hosted cluster must have a unique name in order of {mce-short} to manage it.
```
Comment (lahinson) [inline clusters/hosted_control_planes/managing_hosted_aws.adoc]:
```suggestion
*Note:* Each hosted cluster must have a unique name in order for {mce-short} to manage it.
```

File: clusters/hosted_control_planes/managing_hosted_bm.adoc [hunk 3]
```diff
@@ -1,7 +1,9 @@
 [#hosted-control-planes-manage-bm]
 = Managing hosted control plane clusters on bare metal
 
-You can use the {mce} console to create and manage a {ocp} hosted cluster. Hosted control planes are available as a Technology Preview on bare metal.
+You can use the {mce} console or the hcp command line interface to create a {ocp} hosted cluster. The hosted cluster is automatically imported as a managed cluster. If you wish to disable this automatic import feature, refer to https://github.com/stolostron/rhacm-docs/blob/2.9_stage/clusters/hosted_control_planes/hosted_disable_auto_import.adoc.
```
Comment (lahinson) [inline clusters/hosted_control_planes/managing_hosted_bm.adoc]:
```suggestion
You can use the {mce-short} console or the hosted control plane command line interface (`hcp`) to create an {ocp-short} hosted cluster. The hosted cluster is automatically imported as a managed cluster. If you want to disable this automatic import feature, see xref:../hosted_control_planes/hosted_disable_auto_import.adoc#hosted-disable-auto-import[Disabling the automatic import of hosted clusters into {mce-short}].
```

File: clusters/hosted_control_planes/managing_hosted_bm.adoc [hunk 4]
```diff
@@ -1,7 +1,9 @@
...
+
+Note: Each hosted cluster must have a unique name in order of {mce-short} to manage it.
```
Comment (lahinson) [inline clusters/hosted_control_planes/managing_hosted_bm.adoc]:
```suggestion
*Note:* Each hosted cluster must have a unique name in order for {mce-short} to manage it.
```

File: clusters/hosted_control_planes/managing_hosted_kubevirt.adoc [hunk 5]
```diff
@@ -8,6 +8,10 @@ With hosted control planes and {ocp-virt}, you can create {ocp-short} clusters w
 * Reduces cluster provision time by eliminating the bare metal node bootstrapping process
 * Manages many releases under the same base {ocp-short} cluster
 
+You can use the hcp command line interface to create a {ocp} hosted cluster. The hosted cluster is automatically imported as a managed cluster. If you wish to disable this automatic import feature, refer to https://github.com/stolostron/rhacm-docs/blob/2.9_stage/clusters/hosted_control_planes/hosted_disable_auto_import.adoc.
```
Comment (lahinson) [inline clusters/hosted_control_planes/managing_hosted_kubevirt.adoc]:
```suggestion
You can use the hosted control plane command line interface (`hcp`) to create an {ocp-short} hosted cluster. The hosted cluster is automatically imported as a managed cluster. If you want to disable this automatic import feature, see xref:../hosted_control_planes/hosted_disable_auto_import.adoc#hosted-disable-auto-import[Disabling the automatic import of hosted clusters into {mce-short}].
```

File: clusters/hosted_control_planes/managing_hosted_kubevirt.adoc [hunk 6]
```diff
@@ -8,6 +8,10 @@ With hosted control planes and {ocp-virt}, you can create {ocp-short} clusters w
...
+
+Note: Each hosted cluster must have a unique name in order of {mce-short} to manage it.
```
Comment (lahinson) [inline clusters/hosted_control_planes/managing_hosted_kubevirt.adoc]:
```suggestion
*Note:* Each hosted cluster must have a unique name in order for {mce-short} to manage it.
```

**Review by lahinson — CHANGES_REQUESTED** [review]
@rokej I provided a few edits. Let me know if you have any questions. Thanks!

**Comment by rokej** [comment]
@lahinson Thank you.

**Review by lahinson — APPROVED** [review]

**Pull request merged**